packages/lego-bricks/src/components/Layout/Page/Sidebar.module.css

```
@@ -0,0 +1,34 @@
+.sidebar {
+  position: relative;
+  max-width: calc(var(--lego-max-width) / 4);
+  min-width: 14rem;
+  height: 100%;
+  padding: 1rem;
```

```suggestion
  padding: 1rem 2rem;
```
(although with var(--spacing-thingy))

The sidebar felt a little bit cramped together when the padding outside the container disappeared. Thoughts on adding a little bit more?